config/conference.ts

```diff
@@ -13,7 +13,7 @@ const date = moment.parseZone('2019-08-03T08:00+08:00')
 const endDate = date.clone().add(12, 'h')
 const currentInstance = parseInt(date.format('YYYY'), 10)
 const firstInstance = 2015
-const registrationOpenFrom = moment.parseZone('2019-04-30T17:00:00+08:00')
+const registrationOpenFrom = moment.parseZone('2019-04-01T17:00:00+08:00')
```

Comment:
You shouldn't need to change this, the testing control should allow you to see what the site looks like when registration is open.

Reply:
Will use testing tool next time :) 
Thanks for letting me know